packages/components-native/src/ButtonGroup/ButtonGroup.tsx

```diff
@@ -75,6 +76,7 @@ export function ButtonGroup({
                 fullHeight={true}
                 icon={icon}
                 loading={loading}
+                testID={`primary-action-${index}`}
```

Comment:
We should update this test id to be in the ATL-* pattern: https://jobber.atlassian.net/wiki/spaces/TEAMATL/pages/2476933462/How+should+we+declare+test+ID+s+in+Atlantis+components